config/options.go

```diff
@@ -57,16 +60,36 @@ type Options struct {
 	// This should be used only for testing.
 	InsecureServer bool `mapstructure:"insecure_server" yaml:"insecure_server,omitempty"`
 
-	// Cert and Key is the x509 certificate used to hydrate TLSCertificate
+	// AutoCert enables fully automated certificate management including issuance
+	// and renewal from LetsEncrypt. Must be used in conjunction with AutoCertFolder.
+	AutoCert bool `mapstructure:"autocert" yaml:"autocert,omitempty"`
+
+	// AutoCertHandler is the HTTP challenge handler used in a http-01 acme
+	// https://letsencrypt.org/docs/challenge-types/#http-01-challenge
+	AutoCertHandler func(h http.Handler) http.Handler `hash:"ignore"`
+
+	// AutoCertFolder specifies the location to store, and load autocert managed
+	// TLS certificates.
+	// defaults to $XDG_DATA_HOME/pomerium
+	AutoCertFolder string `mapstructure:"autocert_folder" yaml:"autocert_folder,omitempty"`
```

Comment:
nit: `directory` or `dir` rather than `folder`.  The latter doesn't fit in this context.

Reply:
agree. thanks and good catch.